manifests/artifactory.pp

```diff
@@ -9,9 +9,6 @@
 # * path: fully qualified filepath for the download the file or use archive_path and only supply filename. (namevar).
 # * ensure: ensure the file is present/absent.
 # * url: artifactory download url filepath. NOTE: replaces server, port, url_path parameters.
```

Comment:
Is the note still relevant? Maybe better to do that in the changelog?

Reply:
Fixed.